spec/unit/type/ssh_authorized_key_spec.rb

```diff
@@ -87,7 +87,8 @@
         :'ecdsa-sha2-nistp521',
         :ed25519, :'ssh-ed25519',
         :'ecdsa-sk', :'sk-ecdsa-sha2-nistp256@openssh.com',
-        :'ed25519-sk', :'sk-ssh-ed25519@openssh.com'
+        :'ed25519-sk', :'sk-ssh-ed25519@openssh.com',
+        :'ssh-rsa-cert-v01@openssh.com'
```

Comment:
Can you add the rest of the key types here?

Reply:
done